docs/AADEntitites.md

```diff
@@ -0,0 +1,44 @@
+# Azure Active Directory Entities
+This document describes the configuration of entities create in Azure AD by our [deployment script](../src/deployment/deploy.py)
+
+### OneFuzz Application Registration
+This is the registration of the OneFuzz instance:
```

Comment:
```suggestion
This is the registration of the OneFuzz instance.
```

Reply:
(I suggested `:` previously, but it actually looks weird in context, sorry about that!)